.circleci/config.yml

```diff
@@ -5,7 +5,13 @@ aliases:
   - &build_steps
     - checkout
     - run: java -version
-    - run: ./mvnw clean install -Pci
+    - run: mvn clean install -Dmaven.test.skip=true
```

Comment:
This configuration would actually skip the Unit tests also. We should not skip the UTs.